### PR TITLE
Ft/services section

### PR DESCRIPTION
**Pull request by tuyishimejohnson**
We have added the following changes to be reviewed:
-  We added responsiveness for small and medium screens.
-  We also fixed the font size and the positioning of the text for this section.

### CODE REVIEW

File: src/sections/services.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
 <section>
-    <div class="flex lg:flex-row lg:justify-around md:flex-col pb-24 ">
-        <div class="font-medium lg:text-5xl flex flex-col lg:gap-14 gap-12 lg:pl-0 pl-24 text-4xl lg:pb-0 pb-10">
+    <div class="flex lg:flex-row lg:justify-around md:pb-24 flex-col relative -z-40">
+        <div class="font-medium lg:text-5xl flex flex-col lg:gap-14 md:gap-12 gap-8 lg:pl-0 md:pl-24 md:text-4xl lg:pb-0 pb-10 text-2xl pl-10">
             <p>Brand strategy</p>
-            <p class="hidden lg:block">Corporate Identity &<br> Website development</p>
-            <p class="lg:hidden block">Corporate Identity & Website <br> development</p>
+            <p class="md:hidden lg:block ">Corporate Identity &<br> Website development</p>
```
Comment (IvyMurage) [inline src/sections/services.html:5]: Remove spacing from this line

Comment (IvyMurage) [inline src/sections/services.html:5]: Remove spacing from this line

**Review by KevinRebakure — APPROVED** [review]
Good work guys!

File: src/sections/services.html [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
             <p class="hidden lg:block">Digital marketing &<br> content production</p>
             <p class="lg:hidden block">Digital marketing & content production</p>
             <p>Influencer relations</p>
         </div>
 
-        <div class="relative flex justify-end lg:pr-0 pr-16">
+        <div class="relative flex md:justify-end lg:pr-0 md:pr-16 -z-40 px-10">
             <img src="src/assets/images/green-vector-3.svg" alt="green-shape">
-            <p class="absolute lg:top-44 lg:left-20 lg:w-9/12 w-2/5 right-28 lg:text-xl text-white text-2xl bottom-20">Our strategic approach is a collaborative journey that starts with analysing the initial situation and leads to messages that cut through the information overload. We take care of content production and maintain relationships with multipliers to position your brand in the best possible way. From analysis to implementation, we accompany you every step of the way, ensuring that every measure is effective.</p>
+            <p class="absolute lg:top-40 lg:left-24 lg:w-9/12 md:w-2/5 w-3/5 md:right-28 right-16 lg:text-xl text-white md:text-xl
```
Comment (IvyMurage) [inline src/sections/services.html:16]: This is a big space, remove it as well

**Review by IvyMurage — CHANGES_REQUESTED** [review]
Kindly make these changes, to ensure the codebase is clean

**Pull request merged**